cloud/amazon/ec2.ini

```diff
@@ -0,0 +1,136 @@
+# Ansible EC2 external inventory script settings
```

Comment:
Why is this line here? No modules should be using an ini file for data. If there are common areas for the facts to check, they should be moved to `module_common/ec2.py` as a special param list for ECS. All settings for modules must go through parameters.